docs-web/installation/installation-with-helm.md

```diff
@@ -226,7 +226,7 @@ The following tables lists the configurable parameters of the NGINX Ingress cont
      - The name of the ConfigMap used by the Ingress controller.
      - Autogenerated
    * - ``controller.config.entries``
-     - The entries of the ConfigMap for customizing NGINX configuration.
+     - The entries of the ConfigMap for customizing NGINX configuration. See [ConfigMap resource docs](/nginx-ingress-controller/configuration/global-configuration/configmap-resource/) for the list of supported ConfigMap keys. 
```

Comment:
Link is not rendering correctly, taking a look

Reply:
in the tables, it is no longer makrdown, but reStructuredText.
Here is an example of a link 
```
See `Applying Policies </nginx-ingress-controller/configuration/policy-resource/#applying-policies>`_ for more details.
```